db/seeds.d/050-taxonomies.rb

```diff
@@ -1,5 +1,5 @@
 # some associations shouldn't be set or require special handling.
-skip_associations = [:associated_audits, :audits, :default_users, :hosts,
+skip_associations = [:associated_audits, :audits, :default_users, :hosts, :discovered_hosts
```

Comment:
syntax error, missing comma at the end of the line

Reply:
That's what happens when I send prs at midnight... :man_facepalming: